tools/update-postject.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+# Shell script to update postject in the source tree to the latest release.
+
+# This script must be in the tools directory when it runs because it uses the
+# script source file path to determine directories to work in.
+
+set -ex
+
+cd "$( dirname "$0" )/.." || exit
+
+(
+    rm -rf postject-tmp
+    mkdir postject-tmp
+    cd postject-tmp || exit
+
+    ROOT="$PWD/.."
+    [ -z "$NODE" ] && NODE="$ROOT/out/Release/node"
+    [ -x "$NODE" ] || NODE=$(command -v node)
+    NPM="$ROOT/deps/npm/bin/npm-cli.js"
+
+    "$NODE" "$NPM" init --yes
+
+    "$NODE" "$NPM" install --global-style --no-bin-links --ignore-scripts postject
+)
+
+rm -rf deps/postject
```

Comment:
I see, so the dependency is updated every time the action runs. So this dependency is a "clone" of the original repository. Nice. This clarifies my off-topic question.

Reply:
Correct ✅